praw/models/reddit/live.py

```diff
@@ -8,5 +8,31 @@ class LiveThread(RedditBase):
 
     STR_FIELD = 'id'
 
+    def __eq__(self, other):
+        """Return whther the other instance equals the current."""
+        if isinstance(other, str):
+            return other == str(self)
+        return (isinstance(other, self.__class__) and
+                str(self) == str(other))
+
+    def __hash__(self):
+        """Return the hash of the current instance."""
+        return hash(self.__class__.__name__) ^ hash(str(self))
+
+    def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
+                 _data=None):
+        """Initialize a lazy :class:``LiveThread`` instance.
+
+        :param reddit: An instance of :class:`.Reddit`.
+        :param id: A live thread ID, e.g., ``ukaeu1ik4sw5``
+        """
+        if bool(id) == bool(_data):
+            raise TypeError('Either `id` or `_data` must be provided.')
+        super(LiveThread, self).__init__(reddit, _data)
+        if id:
+            self.id = id  # pylint: disable=invalid-name
+        # else:
```

Comment:
The comment here can be deleted since at the moment there is no where in PRAW's code that is explicitly passing the data attribute -- that's unique to comments.